lib/api_watch_diff.ml

```diff
@@ -120,3 +121,34 @@ let diff_interface ~reference ~current =
     | _, _ -> [ Any ]
     | exception Includemod.Error _ -> [ Any ]
   else value_diffs
+
+let to_diff (diff_result : diff list) : t =
```

Comment:
I'd use `to_text_diff` here. We have our own `diff` type so this name is a bit confusing.